helm/cluster/values.schema.json

```diff
@@ -756,6 +756,12 @@
                         }
                     }
                 },
+                "cgroupsv1": {
+                    "type": "boolean",
+                    "title": "Cgroups v1",
```

Comment:
Please don't tell me it's a capital C because some validation requires this... 🙄🤣

Reply:
Also: Would it make sense, to have this as a string? Just in case something like a v3 is coming one day.

Reply:
It's not like we just put the value of the variable in the configuration, but depending on the boolean value we run some logic to render different things. So I don't think we get much value out of making it a string, If a v3 comes along, we don't know what changes that may require.

Reply:
Sure, sure, I was just thinking of some kind of "switch case" logic on the string then. That would make it more flexible than having to change all the code instead of just adding another `case`.